core/src/jvmMain/kotlin/io/kinference/compiler/generation/operators/DefaultOperatorGenerator.kt

```diff
@@ -0,0 +1,54 @@
+package io.kinference.compiler.generation.operators
+
+import com.squareup.kotlinpoet.MemberName
+import io.kinference.graph.Context
+import io.kinference.operators.Operator
+
+class DefaultOperatorGenerator(
```

Comment:
Please, add base description to understand situations when we should use this operator instead someone else